script.js

```diff
@@ -0,0 +1,48 @@
+let name = prompt("Enter your name");
```

Comment:
Можно использовать `const` если не присваивается другое значение. То же самое в остальных.
```suggestion
const name = prompt("Enter your name");
```